src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
 public class AuthenticationService {
+
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            if (userService.findByEmail(email).getPassword() == password) {
+                return true;
+            }
+        }
         return false;
```

Comment:
we can simplify it by removing the `if`s and returning the result right away by combining 2 conditions into 1 with `&&`

Reply:
do I need to use "else" or I could leave "return false", can not find proper explaining